diff --git a/modules/redirect_domain/src/EventSubscriber/DomainRedirectRequestSubscriber.php b/modules/redirect_domain/src/EventSubscriber/DomainRedirectRequestSubscriber.php
index bc949b8..6f6d4a0 100644
--- a/modules/redirect_domain/src/EventSubscriber/DomainRedirectRequestSubscriber.php
+++ b/modules/redirect_domain/src/EventSubscriber/DomainRedirectRequestSubscriber.php
@@ -85,6 +85,9 @@ class DomainRedirectRequestSubscriber implements EventSubscriberInterface {
         foreach ($domains[str_replace('.', ':', $host)] as $item) {
           if ($this->pathMatcher->matchPath($path, $item['sub_path'])) {
             $destination = $item['destination'];
+            if ($item['sub_path'] == '/*') {
+              $destination .= $path;
+            }
             break;
           }
         }
